db/schema.rb

```diff
@@ -63,7 +63,6 @@
     t.datetime "confirmed_at", precision: 6
     t.datetime "confirmation_sent_at", precision: 6
     t.string "unconfirmed_email"
-    t.index ["confirmation_token"], name: "index_users_on_confirmation_token", unique: true
```

Comment:
hi! @phongvhd93 i am not sure why we have this change here? As we don't have any change in the migration.

Reply:
Yes, that's weird, I didn't touch that file. It seems the system updated the file itself. I will revert this change.

Reply:
`schema.rb` is auto-generated by Rails. It will reflect the current state of the DB based on the migrations that were run. 

This often occurs when you start your project in BRANCH_A (that includes new migrations). Then you switch branches with the project still running, so schema.rb is overwritten by Rails and the new branch has a pending change on `shcema.rb`. If you commit your code too quickly you might include the unwanted changes of `schema.rb` in the new branch — no worries, it happens all the time to everyone :harold: